java/src/com.zeroc.ice/src/main/java/com/zeroc/Ice/Object.java

```diff
@@ -158,6 +176,12 @@ static CompletionStage<OutgoingResponse> _iceD_ice_ids(Object obj, IncomingReque
     }
 
     /**
+     * Dispatches an incoming request and returns the corresponding outgoing response
+     * for the {@code ice_id} operation.
+     *
+     * @param obj the object to dispatch the request to
+     * @param request the incoming request
+     * @return a {@link CompletionStage} that will complete with the outgoing response
```

Comment:
Do we need to add doc comments to "@hidden" methods?

Reply:
Yes. Quite often, I think it's related to inheritance, even though its hidden javadoc still wants a doc comment.